tools/firely-server-ingest.rst

```diff
@@ -279,6 +279,14 @@ Your license already permits the usage of FSI if it contains ``http://fire.ly/vo
 Release notes
 -------------
 
+.. _fsi_releasenotes_2.0.0
+
+Release 2.0.0, January 26th, 2023
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Upgraded to work with the database schemas for :ref:`Firely Server 5.0.0-beta1<vonk_releasenotes_5_0_0-beta1>`
+* 
+
```

Comment:
And it fixes the indexing for versioned references

Reply:
Added